x/concentrated-liquidity/position.go

```diff
@@ -401,3 +439,49 @@ func (k Keeper) validatePositionsAndGetTotalLiquidity(ctx sdk.Context, owner sdk
 	}
 	return basePosition.PoolId, basePosition.LowerTick, basePosition.UpperTick, totalLiquidity, nil
 }
+
+// GetConcentratedLockupDenom returns the concentrated lockup denom for a given pool and position.
+func GetConcentratedLockupDenom(poolId, positionId uint64) string {
+	return fmt.Sprintf("cl/pool/%d/%d", poolId, positionId)
```

Comment:
Can we move this `cl/pool` prefix somewhere more visible? E.g. `keys.go`?

Reply:
yes, moved to cl.go in the types package here https://github.com/osmosis-labs/osmosis/pull/4882/commits/044b4b923a62996cff40bb56caed5d108fbc1597